.stylelintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
           "camelCaseSvgKeywords": true
       }
     ],
-    "font-family-name-quotes": "always-unless-keyword"
+    "font-family-name-quotes": "always-unless-keyword",
+    "declaration-property-value-no-unknown": true
```

Comment:
THIS is what we want. This'll find unknown/invalid values and flag them, such as 

`Unexpected unknown value "space-between" for property "align-items"`
and
`Unexpected unknown value "'auto'" for property "outline"` <-- originally found typo that caused all this